apps/projects/contracts/Projects.sol

```diff
@@ -393,24 +400,27 @@ contract Projects is IsContract, AragonApp {
      * @param _repoId the github repo id of the issue
      * @param _issueNumber the github issue up for assignment
      * @param _submissionAddress IPFS hash of the Pull Request
-     * @param _fulfillmentId retrieved from event after the work is submitted to the bounties contract externally 
+     * //param _fulfillmentId retrieved from event after the work is submitted to the bounties contract externally 
      */
     function submitWork(
         bytes32 _repoId, 
         uint256 _issueNumber, 
-        string _submissionAddress,
-        uint256 _fulfillmentId
+        string _submissionAddress
+        //uint256 _fulfillmentId
     ) external isInitialized
     {
-        require(msg.sender == repos[_repoId].issues[_issueNumber].assignee, "User not assigned to this issue");
-
-        repos[_repoId].issues[_issueNumber].workSubmissions[msg.sender] = WorkSubmission(
+        //require(msg.sender == repos[_repoId].issues[_issueNumber].assignee, "User not assigned to this issue");
+        GithubIssue storage issue = repos[_repoId].issues[_issueNumber];
```

Comment:
Might be better to have this before the require and then check against it instead of pulling the data twice but there's an argument to be made for both ways.